### ISSUE TITLE
Option to use id_ or fw_ URLs

### ISSUE THREAD

**Issue by Sgeo** [atauenis/webone#48]
Can there be options on how the redirects work? e.g. putting id_ at the end of the date in the archive.org URL returns the exact original content (which may refer to further dead stuff, but that too should get redirectd)

**Comment by Sgeo** [comment]
This would probably work quite well with using internal redirects for archive.org

**Comment by atauenis** [comment]
Thank you! Wayback Machine snapshots are looking better in old browsers if Archive.org's toolbar is hidden (`fw_` suffix). This can be enabled via this edit set in configuration file:
```INI
[Edit:^http://web.archive.org/web/([0-9]*)/(.*)]
; Disable Web Archive toolbar on archived pages
IgnoreUrl=webdav
AddInternalRedirect=http://web.archive.org/web/$1fw_/$2
```
This will be included in default configuration in WebOne 0.11.1.

To allow displaying really unmodified pages (with `id_` suffix) it is need to modify Transit.cs file somewhere under 600th line, which I'll do in free time.
